src/dashboard/public/styles.css

```
@@ -22,16 +30,45 @@
-* { margin: 0; padding: 0; box-sizing: border-box; }
+* { box-sizing: border-box; }
 
 html, body {
+    margin: 0;
+    padding: 0;
     height: 100%;
 }
 
 body {
-    font-family: 'Inter', 'SF Pro Display', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
+    font-family: 'Inter Tight', 'Inter', 'SF Pro Display', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.11.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)